.github/README.md

```diff
@@ -4,47 +4,47 @@ _Looking for Umbraco version 8? [Click here](https://github.com/umbraco/Umbraco-
 
 Umbraco CMS
 ===========
-The friendliest, most flexible and fastest growing ASP.NET CMS used by more than 443,000 websites worldwide: [https://umbraco.com](https://umbraco.com)
+The friendliest, most flexible and fastest growing ASP.NET CMS, and used by more than 443,000 websites worldwide: [https://umbraco.com](https://umbraco.com)
 
 [![ScreenShot](img/vimeo.png)](https://vimeo.com/172382998/)
 
 ## Umbraco CMS
-Umbraco is a free open source Content Management System built on the ASP.NET platform. Our mission is to help you deliver delightful digital experiences by making Umbraco friendly, simpler and social.
+Umbraco is a free open source Content Management System built on the ASP.NET platform. Our mission is to help you deliver delightful digital experiences by making Umbraco friendly, simple and social.
```

Comment:
"simpler" is our mission statement, we can't change that :-) 

See: https://umbraco.com/about-us/mission-and-vision/

Reply:
You're not wrong! But.. you know, decisions have been made! 😉